datacenterlight/cms_plugins.py

```diff
@@ -91,13 +91,38 @@ def render(self, context, instance, placeholder):
         context['templates'] = VMTemplate.objects.all()
         context['children_to_side'] = []
         context['children_to_content'] = []
+        pricing_plugin_model = None
+        if instance.child_plugin_instances:
```

Comment:
don't know the exact reason here, but using `if instance.child_plugin_instances` instead of `if instance.child_plugin_instances is not None` passes the condition even when no child plugins are added, leading to an error in the next line.

Reply:
That is not True, I think. But, ok we can make the change.

> `if instance.child_plugin_instances` passes the condition even when no child plugins are added